packages/odyssey-storybook/src/components/odyssey-mui/Checkbox/Checkbox.stories.tsx

```diff
@@ -216,3 +223,27 @@ export const Invalid: StoryObj<CheckboxProps> = {
     checkTheBox({ canvasElement, step })("Checkbox Disabled");
   },
 };
+
+export const Uncontrolled: StoryObj<typeof Checkbox> = {
+  args: {
+    label: "Pre-flight systems check complete",
+    isDefaultChecked: true,
+  },
+};
+
+export const Controlled: StoryObj<typeof Checkbox> = {
+  args: {
+    label: "Pre-flight systems check complete",
+    isChecked: true,
+    onChange: () => {},
+  },
+  render: function C(args) {
+    const [isChecked, setIsChecked] = useState(true);
+    const onChange = useCallback((_, checked) => setIsChecked(checked), []);
+    return (
+      <Checkbox
+        {...{ ...args, isChecked, onChange, isDefaultChecked: undefined }}
+      />
```

Comment:
Why not just pass in the other props :P:

```suggestion
      <Checkbox
        {...args}
        isChecked={isChecked}
        isDefaultChecked={undefined}
        onChange={onChange}
      />
```

Reply:
Oops yea, I was trying to figure this one out in that, `isDefaultChecked` seems like a required prop here, even though `Checkbox` is expecting `isDefaultChecked?: never;` (with `isChecked` being present).